src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  max-width: 1200px;
+  margin: 0 auto;
+  font-size: 12px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  margin: 10px 0;
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  padding-left: 0;
+}
+
+.nav__list li {
+  display: flex;
+  list-style-type: none;
```

Same here, add class for li. Also do not use nesting without need. Fix all similar cases
```suggestion
.nav__list-item {
  display: flex;
  list-style-type: none;
```